nansat/nansat.py

```diff
@@ -1,8 +1,8 @@
 # Name:  nansat.py
 # Purpose: Container of Nansat class
-# Authors:      Asuka Yamakawa, Anton Korosov, Knut-Frode Dagestad,
-#               Morten W. Hansen, Alexander Myasoyedov,
-#               Dmitry Petrenko, Evgeny Morozov, Artem Moiseev
```

Comment:
are these names necessary? It is obvious from the commit history, isn't it? Maybe that's a better way to find authors? However, that question is not really relevant to the issue...